### PR TITLE
AA/attester: add README docs

### LINKED ISSUE #490: How to build attestation-agent/attester/src/bin/evidence_getter.rs to bin to get evidence on AMD snp machine?
How to build attestation-agent/attester/src/bin/evidence_getter.rs to bin to get evidence on AMD snp machine?

### PR DESCRIPTION
**Pull request by Xynnn007**
cc @fffmonkeyking

Close #490

### CODE REVIEW

File: attestation-agent/attester/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## Evidence Getter Tool
+
+This crate provides a simple tool to detect the current platform type and get related quote due to given report data.
+
+Build the binary
+```shell
+git clone https://github.com/confidential-containers/guest-components.git
+cd guest-components/attestation-agent/attester
+
+cargo build --features=bin --bin evidence_getter --release
```
Comment (mythi) [inline attestation-agent/attester/README.md:18]: I had forgotten that I had a slightly patched `evidence_getter`. I believe this needs to be 
```suggestion
cargo build --features=bin,codicon --bin evidence_getter --release
```
maybe `Cargo.toml` needs updating too

Comment (Xynnn007) [reply attestation-agent/attester/README.md:18]: I have test the command in this PR locally and finished. `codicon` is a dependency required by `csv-attester` that is by default enabled by `default-features`

Comment (mythi) [reply attestation-agent/attester/README.md:18]: Indeed, I had only `tdx-attester` enabled through local modifications...

**Review by fitzthum — APPROVED** [review]
Nice

**Pull request merged**